runtime/Cpp/runtime/src/Vocabulary.h

```diff
@@ -179,7 +164,7 @@ namespace dfa {
     /// </param>
     /// <returns> The display name of the token type, for use in error reporting or
     /// other user-visible messages which reference specific token types. </returns>
-    virtual std::string getDisplayName(size_t tokenType) const;
+    std::string getDisplayName(size_t tokenType) const;
```

Comment:
A string_view as result would also make sense here, right?

Reply:
We cannot unfortunately because the implementation uses std::to_string which return std::string. std::string_view is non-owning, so the backing storage for std::string would disappear leading to std::string_view pointing to garbage.